src/client.ts

```diff
@@ -183,6 +183,8 @@ export class Client {
   public start(): void {
     if (this.config.isValid) {
       this.extension.start()
+    } else if (process.env["APPSIGNAL_DIAGNOSE"]) {
+      console.log("Diagnose mode is enabled, not starting extension")
```

Comment:
Should this maybe be logged through the integration logger?
```suggestion
      this.integrationLogger.log("Diagnose mode is enabled, not starting extension")
```
(not sure if this is how you access it, I don't remember)